docs/source/using/sagemaker_notebook_ui.md

```diff
@@ -0,0 +1,57 @@
+## Run Aim UI on SageMaker Notebook instance.
+
+Inject the configuration [script](https://github.com/aimhubio/aim/blob/feature/notebook-extension-sagemaker-support/aim/scripts/sagemaker/on-start.sh) into your notebook instance's lifecycle - `On start`  phase:
```

Comment:
is this the correct link? https://github.com/aimhubio/aim/blob/feature/notebook-extension-sagemaker-support/

Reply:
The link is https://github.com/aimhubio/aim/blob/feature/notebook-extension-sagemaker-support/aim/scripts/sagemaker/on-start.sh